app/lib/features/activities/widgets/session_card.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:core';
+
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/features/home/providers/client_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class SessionCard extends ConsumerWidget {
+  final DeviceRecord deviceRecord;
+
+  const SessionCard({
+    Key? key,
+    required this.deviceRecord,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    bool isVerified = deviceRecord.verified();
+    final fields = [isVerified ? 'Verified' : 'Unverified'];
+    final lastSeenTs = deviceRecord.lastSeenTs();
+    if (lastSeenTs != null) {
+      final dateTime = DateTime.fromMillisecondsSinceEpoch(
+        lastSeenTs,
+        isUtc: true,
+      );
+      fields.add(dateTime.toString());
+    }
+    final lastSeenIp = deviceRecord.lastSeenIp();
+    if (lastSeenIp != null) {
+      fields.add(lastSeenIp);
+    }
+    fields.add(deviceRecord.deviceId().toString());
+    return Card(
+      margin: const EdgeInsets.symmetric(vertical: 1),
+      child: ListTile(
+        leading: isVerified
+            ? Icon(
+                Icons.verified_rounded,
+                color: Theme.of(context).colorScheme.success,
+              )
+            : Icon(
+                Icons.question_mark_rounded,
+                color: Theme.of(context).colorScheme.error,
+              ),
+        title: Text(deviceRecord.displayName() ?? ''),
+        subtitle: Text(fields.join(' - ')),
```

Comment:
that isn't really the proper flutter way of doing this ;) ... 

```suggestion
        subtitle: Row(children: fields.map((x) => Text(x)).toList()),
```

or something along those lines..

Reply:
Fixed by https://github.com/acterglobal/a3/pull/852/commits/e188952180a4a6cfca3a59480ff3a387b0a0c61f